app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from __future__ import annotations
```

Comment:
The `from __future__ import annotations` statement is unnecessary here unless you are using forward references in type hints, which is not the case in this file. You can remove this line if it's not needed.